sklearn/preprocessing/_data.py

```diff
@@ -3394,6 +3394,14 @@ def _more_tags(self):
         return {"allow_nan": True}
 
 
+@validate_params(
+    {
+        "X": ["array-like"],
+        "method": [StrOptions({"yeo-johnson", "box-cox"})],
+        "standardize": ["boolean"],
+        "copy": ["boolean"],
```

Comment:
```suggestion
```

Reply:
I've reflected your suggestions. Thanks for the help!